examples/pubspec.yaml

```diff
@@ -0,0 +1,49 @@
+name: flame_example
+description: A set of small game examples showcasing each feature provided by the Flame Engine.
+publish_to: 'none'
+
+version: 0.1.0-rc2
```

Comment:
We could remove the -rc2 part here I think, and just never bother to update this version since it is not publishable

Reply:
pub complains if a non-rc pck references a rc one. but I will change the reference to be a local path